lib/src/services/gamepad.dart

```diff
@@ -0,0 +1,42 @@
+import "package:win32_gamepad/win32_gamepad.dart";
+
+import "service.dart";
+
+///A service to send the comtroller's activity
+///using win32_gamepad
+class GamepadService extends Service {
+  ///The connection to the controller
+  static final gamepad = Gamepad(0);
+  
+  @override
+  Future<void> init() async{ 
+    connect();
+  }
```

Comment:
```suggestion
  @override
  Future<void> init() async { }
```
See my comment below -- it seems `connect` doesn't actually do anything so we should get rid of it. Then there's nothing to do when initializing!